src/lumigo_tracer/spans_container.py

```diff
@@ -150,13 +150,12 @@ def add_unparsed_request(self, parse_params: HttpRequest):
         """
         if self.http_spans:
             last_event = self.http_spans[-1]
-            if last_event and last_event.get("type") == HTTP_TYPE:
+            if last_event and last_event.get("type") == HTTP_TYPE and self.previous_request:
```

Comment:
What happens if this is the first request? We won't have `previous_request`, isn't it?

Reply:
excatly, and then we don't need to do this part.
Previously we handle it be giving default parameters that doesn't change anything - and I think that. this is better approach :)